views/apartments/edit.erb

```diff
@@ -0,0 +1,13 @@
+<h2>Edit <%= @apartment.address %> </h2>
+
+<form method="post" action="/apartments/<%=@apartment.id%>">
+  <ul>
+  <input type="hidden" name="_method" value="put">
```

Comment:
Similar to my below comment, would move this before the `<ul>` but keep it inside the `<form>`.